languages/cpp/README.md

```diff
@@ -77,14 +78,29 @@ SecretIdentifiersResponse secretIdentifiersResponse = bitwardenClient.listSecret
 
 ### Get secret details
 
-```
+```c++
 boost::uuids::uuid secretId = boost::uuids::string_generator()(secretResponseCreate.get_id());
```

Comment:
⛏️ Similarly as my other comment, we can extract the `boost::uuids::uuid secretId` to be part of `### Add new secret`

Reply:
Addressed in 7ab21345d0d91c7dd4a152ef7b33da507f2bae6b.